dependencies.props

```diff
@@ -0,0 +1,92 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+  <PropertyGroup>
+    <CoreFxCurrentRef>996189e56e6cc7e6f71a371e18efde7d4f8e104f</CoreFxCurrentRef>
+    <CoreClrCurrentRef>5ceb13b6c4b9b6b95c0b77b59c0477475e7bba54</CoreClrCurrentRef>
+    <ExternalCurrentRef>cdb8679634e573847e20c1eae0fa3918f5daf1e6</ExternalCurrentRef>
+    <WcfCurrentRef>8373288e4fc75df9f37e54f8d6bab15d0e07a083</WcfCurrentRef>
+  </PropertyGroup>
+
+  <!-- Auto-upgraded properties for each build info dependency. -->
+  <PropertyGroup>
+    <CoreFxExpectedPrerelease>beta-24512-01</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>beta-24512-03</CoreClrExpectedPrerelease>
+    <WCFExpectedPrerelease>beta-24431-01</WCFExpectedPrerelease>
```

Comment:
From just searching in github it looks like only `CoreFxExpectedPrerelease` is used--if that's true, you can remove the other two. (And their `XmlUpdateStep`s.)